### PR TITLE
Bump to Git 2.22.0

### LINKED ISSUE #264: update to Git 2.22
Git 2.22 is slated to be released June 7th, based on [the calendar](http://tinyurl.com/gitcal). The Git for Windows packages should be available a day or so after that.

### PR DESCRIPTION
**Pull request by niik**
closes #264 

- [x] confirm all builds green
- [ ] tag release
- [ ] :shipit:

### CODE REVIEW

**Comment by ceciltoddf** [comment]
dont no

**Comment by ceciltoddf** [comment]
what do i need to do next

**Comment by arekzaluski** [comment]
Is there anything blocking this release?

**Comment by billygriffin** [comment]
I think this PR is now superseded by #316 and this and the associated issue should be closed unless I'm incorrect?

**Comment by niik** [comment]
> I think this PR is now superseded by #316 and this and the associated issue should be closed unless I'm incorrect?

You are correct!

File: dependencies.json [hunk 1]
```diff
@@ -1,20 +1,20 @@
 {
```
Comment (chris20202003) [inline dependencies.json:1]: Duplicate of"bd91db55bd95eaa80687df28877e2df8c8858a0266e9c67331cfddba2735f25c"

Comment (chris20202003) [reply dependencies.json:1]: "filename": "MinGit-2.22.0-64-bit.zip",

**Pull request closed without merging**